axelor-base/src/main/java/com/axelor/apps/base/service/RegistrationNumberValidationDefault.java

```diff
@@ -0,0 +1,14 @@
+package com.axelor.apps.base.service;
+
+import com.axelor.apps.base.db.Partner;
+
+public class RegistrationNumberValidationDefault extends RegistrationNumberValidation {
+  public boolean computeRegistrationCodeValidity(String registrationCode) {
```

Comment:
Please add `@Override` annotation here:  https://google.github.io/styleguide/javaguide.html#s6.1-override-annotation

same issue for the other subclass